submit_molsim_job.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#SBATCH --job-name=my_molsim_job         # Name des Jobs
+#SBATCH --output=my_molsim_job_out.txt    # Datei für Standardausgabe
+#SBATCH --error=my_molsim_job_err.txt     # Datei für Fehlerausgabe
+#SBATCH --time=04:00:00                  # Maximale Laufzeit (Stunden:Minuten:Sekunden)
+#SBATCH --clusters=serial                # Cluster wählen (hier: serial)
+#SBATCH --partition=serial_std           # Partition wählen (hier: serial_std)
+#SBATCH --mem=4gb                        # Speicher pro Job
+#SBATCH --cpus-per-task=1                # Anzahl der CPUs pro Job
+#SBATCH --mail-type=END                  # E-Mail-Benachrichtigung bei Jobende
+#SBATCH --mail-user=ge47jow@tum.de # Ihre E-Mail-Adresse für Benachrichtigungen
+
+# Laden der benötigten Module
+module load gcc
+module load cmake
+
+# Navigieren zum Verzeichnis, in dem das Programm kompiliert wird
+
+mkdir build
+cd build
+
+# Kompilieren des Programms mit gprof
+cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-fsanitize=address" ..
+make
```

Comment:
you don't need to do this on compute node. Just do this on the login node, and in the script provide the command with path to executable to run.